docs/spark-qualification-tool.md

```diff
@@ -380,7 +380,7 @@ Here is a brief description of each of column that is in the CSV:
 15. Unsupported Write Data Format: Reports the data format which we currently don’t support, i.e.
     if the result is written in JSON or CSV format.
 16. Complex Types: Looks at the Read Schema and reports if there are any complex types(array, struct or maps) in the schema.
-17. Unsupported Nested Complex Types: Nested complex types are complex types which
+17. Nested Complex Types: Nested complex types are complex types which
```

Comment:
actually if we aren't saying these are unsupported do we need more text here stating why we are reporting these separately?  Should we say something like not all nested types supported, please see the docs for the release you are using?
@viadea ideas here to make sure its clear why this field is here?

Reply:
I think we do claim that there is 
```
Here the Problematic Duration is zero but please keep in mind that we are only able to detect certain issues. This currently includes some UDFs, some decimal operations and nested complex types. 
```
So I think this should be fine.

Reply:
@tgravescs Please let me know if you think more context is required here in the docs. Or the one which @viadea has posted would be sufficient.

Reply:
ok its fine.